README.md

```diff
@@ -33,7 +33,7 @@ fn action() -> Result<u64, ()> {
 fn main() {
     let mut core = Core::new().unwrap();
 
-    let retry_strategy = ExponentialBackoff::from_millis(10)
+    let retry_strategy = ExponentialBackoff::from_millis(10, None)
```

Comment:
Instead of changing the constructor, should we introduce a second `from_millis_with_max` constructor instead? Basically following progressive disclosure of complexity.